doc/appendix-de-kassensichv/appendix-de-kassensichv.md

```diff
@@ -1,3 +1,3 @@
 # Appendix: DE - KassSichV
 
-This appendix expands on the information provided in the General Part by adding details specific to the German market. This additional information is provided only where applicable. The remaining chapters, for which there is no additional information required, were omitted.
+This appendix expands on the General Part's information by adding details specific to the German market. This additional information is provided only where applicable. The remaining chapters, for which there is no further information required, were omitted.

```

Comment:
I liked it better the way it was before :D so information provided in the General Part